saleor/graphql/discount/filters.py

```diff
@@ -154,7 +154,7 @@ def filter_start_date_range(qs, _, value):
 
     @staticmethod
     def filter_is_old_sale(qs, _, value):
-        return qs.filter(old_sale=value)
+        return qs.exclude(old_sale_id__isnull=value)
```

Comment:
Shouldn't this be: `qs.exclude(old_sale_id__isnull=True)`? And it doesn't need a `value` passed in this case, I think?

Reply:
With `value=False` passed you can get promotion set which doesn't derive from old sale. I think it can be useful.

Reply:
Ah, makes sense, for some reason I thought value is something else than a bool 😅